Source/Controllers/DataControllers/SPDatabaseStructure.m

```diff
@@ -286,12 +286,13 @@ - (void)queryDbStructureWithUserInfo:(NSDictionary *)userInfo
 #endif
 
 		// Delete all stored data for the database to be updated, leaving the structure key
-		[queriedStructure removeObjectForKey:db_id];
+		[queriedStructure safeRremoveObjectForKey:db_id];
```

Comment:
I assume this should be remove and nor rremove?

Reply:
ahhhh